config/manager/manager.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 kind: Namespace
 metadata:
   labels:
-    control-plane: controller-manager
+    control-plane: infra-controller-manager
```

Comment:
I'm not sure what this renaming does, and whether it has to match what is defined in config/default/manager_webhook_patch.yaml?
```
metadata:
  name: controller-manager
  namespace: system
```

Reply:
This change is for the label, which does not pertain to the `metadata.name`.  We are repeating this pattern of altering the label `control-plane: controller-manager` to something more specific like `control-plane: infra-controller-manager` because this label is used by the webhook service to direct CR API traffic to the proper webhook endpoint.  Without this change, all operator webhook servers have an identical label, and k8s gets confused and routes traffic for CRs to the wrong associated `controller-manager` webhook server.

Reply:
Oh alright. Thanks for the explanation.